src/client/test/unit/PacWrapper.test.ts

```diff
@@ -33,13 +33,14 @@ class MockPacInterop implements IPacInterop {
 describe('PacWrapper', () => {
     it('AuthList parses correctly', async () => {
         const interop = new MockPacInterop();
-        interop.executeReturnValue = "{\"Status\":\"Success\",\"Errors\":[],\"Information\":[\"Input commands: [\\\"auth\\\",\\\"list\\\"]\",\"Profiles (* indicates active):\"],\"Results\":[{\"Index\":1,\"IsActive\":true,\"Kind\":\"CDS\",\"Name\":\"cctest\",\"Resource\":\"https://contoso.example.com\",\"User\":\"bob@contoso.com\",\"CloudInstance\":\"Public\"}]}";
+        interop.executeReturnValue = "{\"Status\":\"Success\",\"Errors\":[],\"Information\":[\"Input commands: [\\\"auth\\\",\\\"list\\\"]\",\"Profiles (* indicates active):\"],\"Results\":[{\"Index\":1,\"IsActive\":true,\"Kind\":\"CDS\",\"Name\":\"cctest\",\"ActiveOrganization\":{\"EnvironmentUrl\":\"https://contoso-mock.crmtest.dynamics.com\",\"FriendlyName\":\"\",\"User\":\"bob@contoso.com\",\"CloudInstance\":\"Public\"}}]}";
```

Comment:
Looks like the new closing `}` is in the wrong place - should be after the ActiveOrganization's object